src/pages/certificates/create-certificate/create-certificate.js

```diff
@@ -78,7 +78,7 @@ const CreateCertificate = () => {
       }
     });
 
-    if (!isInvalid && check) {
+    if (!isInvalid && check && email) {
```

Comment:
What if I'll type only white spaces? Or any not-email value?

Reply:
We use regular expressions for check.